ci/install_abseil.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Copyright The OpenTelemetry Authors
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+export DEBIAN_FRONTEND=noninteractive
+
+if ! type cmake > /dev/null; then
+    #cmake not installed, exiting
+    exit 1
+fi
+export BUILD_DIR=/tmp/
+export INSTALL_DIR=/usr/local/
+pushd $BUILD_DIR
+git clone --depth=1 -b 20210324.0 https://github.com/abseil/abseil-cpp.git
```

Comment:
Is `20210324.0` a tag name or branch name? And define it in a shell variable?

Reply:
It's a tag name, Made it a shell variable.

Reply:
I think one good option, or at least optimization opportunity, would be to see how gRPC is installing it. Because then we can just anchor to the same exact Abseil used by gRPC/protobuf/OTLP, to minimize code / workspace size... That'll be though non-ABI-portable min-size static build of SDK.

Reply:
Yes, the plan is to reuse Abseil from gRPC install once we add these tests for otlp expoter. As of now, it is not testing otlp exporter.